boards/riscv/hifive_unleashed/hifive_unleashed_defconfig

```diff
@@ -0,0 +1,16 @@
+CONFIG_SOC_SERIES_RISCV_SIFIVE_FREEDOM=y
+CONFIG_SOC_RISCV_SIFIVE_FU540=y
+CONFIG_BOARD_HIFIVE_UNLEASHED=y
+CONFIG_RISCV_MACHINE_TIMER=y
+CONFIG_PLIC=y
+CONFIG_PINMUX=y
```

Comment:
I'm getting the following warning when building either `hifive_unleashed` or `hifive_unmatched`:
```
CMake Warning at ../../CMakeLists.txt:718 (message):
  No SOURCES given to Zephyr library: drivers__pinmux

  Excluding target from build.
```

Is this intentional?

Reply:
Oh, not intentional. I'll fix it. Thanks!

Reply:
Fixed.